documentation/how-to/pg-service.en.md

```diff
@@ -1,96 +1,151 @@
 ---
-title: PostgreSQL service
+title: PostGIS service
 tx_slug: documentation_how-to_pg-service
 ---
+# Working with PostGIS
 
-# PostgreSQL service
+## PostGIS in QField
 
-A `pg_service.conf` file allows to use an named alias for a PostgreSQL server connection.
-Instead of storing hostname, port, database name and more into the QGIS Project file, these can be stored separately.
-It is even possible to store username and password in a `pg_service.conf` file, to avoid having this stored in clear text in the QGIS Project.
+Working with databases can be much easier when working in a multi-user environment or when working in a variety of projects that require more complex data structures.
+With QField you can easily work within a database but there are a few steps that need to be taken before the smoothless fieldwork can begin.
+This page goes through the different options and gives step-by-step instructions on how this can be achieved.
+
+## Connection to PostGIS in QGIS
+
+In QGIS there are two options in which you can connect to your database.
+
+1. **Direct Connection:** When connecting to a PostGIS database, you can store all information including the credentials inside the QGIS Project directly.
+2. **Using a PG Service File:** Using a service file that holds the connection parameters to all your databases as a an individual service name.
+Independently on working with our without QFieldCloud, we highly recommend to make use of this option due to data safety.
```

Comment:
Perhaps clarify that we won't be explaining option 1 in this page. See the next review comment.